.dockerignore

```diff
@@ -0,0 +1,4 @@
+.venv
+.idea
+Dockerfile
+docker-compose.yaml
```

Comment:
You shouldn't put these files to dockerignor. They are necessary to build the container

Reply:
Also add new line at the end of file